SPEC.md

```diff
@@ -33,6 +33,8 @@ Released versions of the spec are available as Git tags.
 |        |   | Add `AdditionalGIDs` to `ContainerEdits` |
 | v0.8.0 |   | Remove .ToOCI() functions from specs-go package. |
 
+*Note*: spec loading fails on unknown fields and when minimal required version is higher than the version specified in the spec. Minimum required version is determined based on the usage of fields mentioned in the table above. For example minimal required version is v0.6.0 if `Annotations` field is used in the spec, but `IntelRdt` is not. Golang zero default values are used for all new fields so the spec remains valid when loading older spec versions.
```

Comment:
```suggestion
*Note*: spec loading fails on unknown fields and when the minimum required version is higher than the version specified in the spec. The minimum required version is determined based on the usage of fields mentioned in the table above. For example the minimum required version is v0.6.0 if the `Annotations` field is used in the spec, but `IntelRdt` is not.

 *Note*: Golang zero default values are used for all new fields so that the spec remains valid when loading older spec versions.
```

Reply:
Also, do we want to reference the `MinimumRequiredVersion` function that we expose -- at least by name?

Reply:
@elezar Done. PTAL again, thanks!